support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -92,27 +98,44 @@ export function CheckoutNudgeContainer({
 
 	let title, subtitle, paragraph;
 
-	if (dynamic) {
+	const calcWeeklyAmount = (annualAmount: number, toNearest?: number) => {
+		const weeklyAmount = Math.ceil((annualAmount * 100) / 52);
+
+		// Current/control solution is to round to the nearest 10p so this may be
+		// disposable if the variant is adopted as we want the dynamic version to be
+		// rounded to the nearest 1p
+		const roundedWeeklyAmount =
+			toNearest !== undefined
+				? Math.round(weeklyAmount / toNearest) * toNearest
+				: weeklyAmount;
+
+		return countryGroupId === 'GBPCountries' && roundedWeeklyAmount < 100
+			? roundedWeeklyAmount.toString() + `p`
+			: currencyGlyph +
+					(roundedWeeklyAmount / 100).toFixed(
+						roundedWeeklyAmount % 100 === 0 ? 0 : 2,
+					);
+	};
+
+	if (isDynamicUs) {
 		title = 'Make it annual';
 		subtitle = `change to ${clampedAmountToCurrenyStr} per year`;
 		paragraph =
 			'Regular, reliable funding from readers is vital for our future. Help protect our open, independent journalism long term.';
 	} else {
-		const minAmount = getConfigMinAmount(countryGroupId, 'ANNUAL');
-		const weeklyMinAmount =
-			Math.round(Math.ceil((minAmount * 100) / 52) / 10) * 10;
-		const minWeeklyAmount =
-			countryGroupId === 'GBPCountries' && weeklyMinAmount < 100
-				? weeklyMinAmount.toString() + `p`
-				: currencyGlyph +
-				  (weeklyMinAmount / 100)
-						.toFixed(weeklyMinAmount % 100 === 0 ? 0 : 2)
-						.toString();
+		if (isDynamicGlobal) {
+			subtitle = `with ${clampedAmountToCurrenyStr} (${calcWeeklyAmount(
+				clampedAmount,
+			)} per week)`;
```

Comment:
Is it possible to avoid reassigning the `title`/`subtitle`/`paragraph` variables here? It would be nice to be able to have a separate function to handle this logic that always returned an object like this:
```
{
	title: 'xyz',
	subtitle: 'abc',
	paragraph: 'lorem impsum',
}
```
than to have `let` variables with their values changing and a possible `undefined` state.

Reply:
It's totally possible, just didn't think it was worth it at this point when they're conditional on A/B tests that are only temporary and when they end, all that conditional logic will presumably be removed and those variables can go back to being simple `const`s that are set once outside of all these if statements

Reply:
That said the outcome of the A/B tests might be that we have one title for US and one for everyone else in the world so it's probably still worth implementing your suggested solution

Reply:
It's been my experience fairly consistently that anything you write 'just for an A/B test' that's a little bit messy has about a 75% chance of ending up being shipped with no time to make it any better. Always better to go for the cleaner version if at all possible.